pkgs/test_core/lib/src/runner/spawn_hybrid.dart

```diff
@@ -86,7 +86,7 @@ StreamChannel spawnHybridUri(String url, Object? message, Suite suite) {
 /// scheme.
```

Comment:
We also need to update this documentation

Reply:
The documentation already says:

"If it's a package: URL, it will be resolved using the current package's dependency constellation."

I can add more detail if that would be helpful but the intent of this PR was to make that statement true ;-)